gulpfile.ts

```diff
@@ -22,14 +24,33 @@ interface APIModel extends GithubObject {
   outputPath: string
 }
 
+// configure dotenv and get github token
+dotenv.config()
+const GITHUB_TOKEN = env.get('GITHUB_TOKEN').required().asString()
+
 // Using authentication to increases Github API rate limit.
-const octokit = new Octokit()
+const octokit = new Octokit({ auth: GITHUB_TOKEN })
```

Comment:
If it uses a personal token, then we cannot use it, as it is not secure.

Can it use a token that is auto generated by GitHub Actions?

Reply:
> Can it use a token that is auto generated by GitHub Actions?

Yes. It can. 

Commit: https://github.com/ScaleLeap/selling-partner-api-sdk/pull/30/commits/732bbe3dfd583dda1525697ac19f6685680fd23a
Log: https://github.com/ScaleLeap/selling-partner-api-sdk/runs/2163812928?check_suite_focus=true

Reply:
Great. Let's do it that way then! :)